### PR TITLE
Added Phone Reminder Bar

### PR DESCRIPTION
**Pull request by CaptainMagma**

### CODE REVIEW

File: resource/layout/phonereminderbar.layout [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+			render {}
+			render_bg {
+				0="image_scale( x0+3, y0, x1-3, y1+1, graphics/emailreminder_center )"
+				1="image      ( x0 + 1, y0 + 1, x0 + 4, y1 + 1, graphics/emailreminder_left   )"
+				2="image      ( x1 - 4, y0 + 1, x1    , y1 + 1, graphics/emailreminder_right  )"
+
+				3="fill(x0 + 1, y0 + 50, x0 + 2, y1 + 1, ScreenB)"
+				4="fill(x0 + 2, y0 + 50, x0 + 3, y1 + 1, Black)"
+				5="fill(x1 - 2, y0 + 50, x1 - 1, y1 + 1, ScreenB)"
+				6="fill(x1 - 3, y0 + 50, x1 - 2, y1 + 1, Black)"
```
Comment (neico) [inline resource/layout/phonereminderbar.layout:48]: Those 4 "fill" need to be removed as per emailreminderbar.layout

File: resource/layout/phonereminderbar.layout [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	styles
+	{
+		CPhoneReminderBar
+		{
+			
+			render {}
+			render_bg {
+				0="image_scale( x0+3, y0, x1-3, y1+1, graphics/emailreminder_center )"
+				1="image      ( x0 + 1, y0 + 1, x0 + 4, y1 + 1, graphics/emailreminder_left   )"
+				2="image      ( x1 - 4, y0 + 1, x1    , y1 + 1, graphics/emailreminder_right  )"
```
Comment (neico) [inline resource/layout/phonereminderbar.layout:43]:
```
                0="image_scale( x0 + 4, y0 + 1, x1 - 4, y1 + 2, graphics/emailreminder_center )"
                1="image     ( x0 + 1, y0 + 1, x0 + 4, y1 + 1, graphics/emailreminder_left   )"
                2="image     ( x1 - 4, y0 + 1, x1    , y1 + 1, graphics/emailreminder_right  )"
```

as per emailreminderbar.layout

File: resource/layout/phonereminderbar.layout [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+			textcolor=white
+		}
+	}
+	
+	colors
+	{
+		phonereminder_bar_green1 "92 126 16 255"
+		phonereminder_bar_green2 "73 100 13 255"
+		outline_gray "77 75 72 255"
+		outline_dark "23 22 20 255"
```
Comment (neico) [inline resource/layout/phonereminderbar.layout:91]: those 4 lines should be removed as per emailreminderbar.layout

**Comment by CaptainMagma** [comment]
@neico could you please explain in greater detail why they need to be removed? I put them there because at the time, I didn't understand how to make the border stretch farther than the height of the emailreminderbar.

**Comment by neico** [comment]
Which are you referring to? the color ones?

Aynway, surely there was a reason to why things differ from the steam default (it's a skin after all), here's the relevant diff:

**Comment by CaptainMagma** [comment]
@neico Those three fill lines are not identical to the ones in the phonereminderbar.layout file because I replaced them with the PixelVision style border. Have you seen what the reminder bar looks like without those four lines? The PixelVision style gray and black border that surrounds it is incomplete on the bottom left and bottom right sides. Test it for yourself so you can see. I added that border because I didn't know if there was any other way to make the border stretch to the height of the reminder bar.

**Comment by somini** [comment]
Sorry for taking so long to answer this, I was away for this past few days.

I tested the last version, and it seems good to me. What's the actual conflict here?

**Comment by CaptainMagma** [comment]
Valve added a phone reminder bar in one of their latest Steam updates. Basically, it's a green pop-up that reminds you that you should add your phone number to Steam for extra security measures. I added and updated the files necessary in order for the pop-up to appear the way it should. Previously it appeared like this: http://i.imgur.com/3OU9m3x.png (image from bug report in PixelVision group discussions).

**Comment by somini** [comment]
Yeah, I got that today and (hapilly :D) verified there was a patch already available.
I was just asking if the discussion with @neico has reached a conclusion, it kinda ends abruptly...

**Comment by somini** [comment]
Well, I tested it and it works, so I'm merging this, people are complaining loudly in that thread.

If @neico has any more comments, feel free to comment on this issue.

**Pull request merged**